Sources/TuistSupport/Logging/Logger.swift

```diff
@@ -53,6 +57,9 @@ public enum LogOutput {
             handler = StandardLogHandler.self
         case .json:
             handler = JSONLogHandler.self
+        case .quiet:
+            LoggingSystem.bootstrap(quietLogHandler(label:))
```

Comment:
```suggestion
            LoggingSystem.bootstrap(quietLogHandler)
```

nit: I think this reads a bit nicer